generators/upgrade/versions.js

```diff
@@ -0,0 +1,107 @@
+module.exports = [
```

Comment:
Do we need to update this file everytime a version released?
I feel like we a doc somewhere to know what to change in the generators when a new version comes out